device/send.go

```diff
@@ -317,6 +319,28 @@ top:
 				goto top
 			}
 
+			if peer.constantPacketSize {
+				mtu, err := peer.device.tun.device.MTU()
```

Comment:
Perhaps you could create `mtu` at the for-loop scope, and assign it once, the first time it's required to be known.

Reply:
Then the padding would stop to be constant size if the mtu changes, which is probably not ideal. However, after looking into it I realized that there is a much cheaper way to get the mtu, just load the always up to date atomic variable.

Reply:
No, the value would then go out of scope when all staged packets have been sent. But your current solution is better.